Doc/whatsnew/3.14.rst

```diff
@@ -135,6 +135,9 @@ symtable
 
   (Contributed by Bénédikt Tran in :gh:`120029`.)
 
+typing
+------
+* Add :func:`~typing.copy_kwargs` to *typing* that copies/applies the ParameterSpec from one function to another.
```

Comment:
We can link to the module, and please wrap at 80 chars: https://devguide.python.org/documentation/markup/#use-of-whitespace

```suggestion
* Add :func:`~typing.copy_kwargs` to :mod:`typing` that copies/applies the
  ParameterSpec from one function to another.
```

Should ParameterSpec be ParamSpec? If so, let's use ``` :class:`~typing.ParamSpec` ```. Or plain text "parameter spec"? Same applies to the NEWS file.